hack/pre-release.sh

```diff
@@ -175,40 +305,16 @@ if [[ $stream != "x" && $stream != "y" && $stream != "z" ]]; then
   echo "input stream argument is $stream"
   echo ""
   usage
-  exit 
+  exit
 fi
 
-# check if the git branch is dirty, and exit if it is
-check_git_branch
-
 initial_branch=$(git branch --show-current)
-new_branch=$(create_git_branch_list $stream-stream-bump $base_branch)
-switch_git_branch $base_branch $new_branch
-
-# Locate and store the WMCO version from the makefile 
-# finds "WMCO_VERSION ?=" and selects text following it
+git checkout $base_branch
+git submodule update --init --recursive  > /dev/null 2>&1
 WMCO_VERSION=$(sed -n -e 's/^.*WMCO_VERSION ?= //p' Makefile)
-update_WMCO_version $stream $WMCO_VERSION
-update_OCP_version $stream $WMCO_VERSION 
-
-# check to see if the opeator-sdk version is correct 
-operator_sdk_current_version=$(operator-sdk version |  grep -oP 'operator-sdk version: "\K[^"]+' )
-if [[ "$operator_sdk_current_version" != "$OPERATOR_SDK_VERSION" ]]; then
-  echo "operator-sdk $OPERATOR_SDK_VERSION must be installed."  
-  echo "exiting."
-  git checkout $initial_branch
-  exit 
-fi
-
-make bundle
 
-commit_message="[$base_branch] Update version to $WMCO_VERSION
+github_update $base_branch $stream $WMCO_VERSION
+dist_git_update $stream $WMCO_VERSION
```

Comment:
do we think it should be useful to be able to run just the dist-git update portion or just the github update portion?

Reply:
Wouldn't that be a requirement? We need the GitHub update merged first before we make dist-git changes.

Reply:
I was thinking that if they were all run at the same time, you wouldn't  forget a step. Since it's making a patch, it's your job to apply and push at the right time.

Reply:
A user should be able to run the two workflows separately IMO